ee/clickhouse/views/experiments.py

```diff
@@ -1,11 +1,13 @@
 from typing import Any, Type, Union
 
+from django.utils.timezone import now
 from rest_framework import serializers, viewsets
 from rest_framework.decorators import action
 from rest_framework.exceptions import ValidationError
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.request import Request
 from rest_framework.response import Response
+from statshog.defaults.django import statsd
```

Comment:
nit: Use prometheus metrics, or you'll only get them on US and not EU

Reply:
hmm although I just added another callsite for an existing statsd metric. will leave as is, and then in a follow-up refactor maybe move the metric to prometheus